boards/arm/stm32/stm32f411-minimum/scripts/Make.defs

```diff
@@ -0,0 +1,100 @@
+############################################################################
+# boards/arm/stm32/stm32f411-minimum/scripts/Make.defs
+#
+#   Copyright (C) 2016-2017 Gregory Nutt. All rights reserved.
+#   Author: Gregory Nutt <gnutt@nuttx.org>
```

Comment:
You can change the license to Apache since Greg is the only copyright holder and author.